Cesium.Runtime/StdIoFunctions.cs

```diff
@@ -23,6 +52,39 @@ public static void PutS(byte* str)
             // return EOF; // TODO[#156]: Uncomment
         }
     }
+    public static int PutChar(byte character)
+    {
+        try
+        {
+            Console.Write((char)character);
+            return 0;
+        }
+        catch (Exception) // TODO[#154]: Exception handling.
+        {
+            const int EOF = -1; // TODO[#155]: Extract to some common place.
+            return EOF;
+        }
+    }
+
+    public static int PutC(byte character, void* stream)
+    {
+        try
+        {
+            var streamDescriptor = GetStreamHandle(stream);
+            if (streamDescriptor == null)
+            {
+                return -1;
+            }
+
+            streamDescriptor.Writer!.Write((char)character);
+            return 0;
```

Comment:
Should be same as `putchar`, basically.

```suggestion
            return character;
```